### DISCUSSION TITLE
Web site credentials

### DISCUSSION THREAD

**Discussion by stevland** [rocksdanister/lively#1479] [Q&A]
Hi Dani,

I absolutely love this project!

I am using Lively to display a website that requires a login. Every day I have to re-enable keyboard and sign into the site.

I have tried both web CefSharp and WebView2... neither of them offer to remember credentials like a full browser would.

Is there any way to interact with either of the browsers and get them to cache my credentials?

At least that way I would only have to click on button each time.

I found Settings.json and set  `"CefDiskCache": true`, but that exhausted my expertise and didn't change anything. ;)

**Comment by rocksdanister** [comment]
Try `WebView2` and enable `Disk cache` in wallpaper settings:
<img width="647" alt="image" src="https://user-images.githubusercontent.com/17554161/205816651-1cba9008-6c65-4248-87bf-240d779ef319.png">
